src/main/kotlin/org/rust/ide/refactoring/generateConstructor/GenerateConstructorHandler.kt

```diff
@@ -90,30 +126,30 @@ data class ConstructorArgument(
     val type: RsTypeReference?
 ) {
 
-    val dialogRepresentation: String get() = "$argumentIdentifier: ${type?.text ?: "()"}"
+    val dialogRepresentation: String get() = "$argumentIdentifier: ${typeReference ?: "()"}"
```

Comment:
```suggestion
    val dialogRepresentation: String get() = "$argumentIdentifier: $typeReference"
```

Also, let's rename `typeReference` to `typeReferenceText` to avoid possible misleading